sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Helpers/RemovableDeclarationCollector.cs

```diff
@@ -147,13 +143,5 @@ private static SyntaxNodeSymbolSemanticModelTuple SelectNodeTuple(SyntaxNode nod
             return container.SyntaxNode.DescendantNodes(IsNodeContainerTypeDeclaration)
                 .Where(node => kinds.Contains(node.Kind()));
         }
-
-        private static bool IsSymbolNotInInterfaceAndRemovable(ISymbol symbol, Accessibility maxAccessibility)

```

Comment:
It would help the review process if you could add context to some of the changes e.g. _Not related to the current bug fix. IsRemovable already checks IsInterface etc so IsSymbolNotInInterfaceAndRemovable is redundant._

Reply:
point taken